.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Fixes #. A short summary which serves as a squashed-commit message.
```

Comment:
`(Fixes | Implements | Optimises) #` or, more general, `Solves #`.

Reply:
Thanks, will change to `Solves #`, I was in search for a proper word :)

I'm also thinking of moving this to the very end. Does anybody have any objections?